misc/example/test.cpp

```diff
@@ -90,9 +86,9 @@ void foo() {
 
   // print the value of A and B.
   for (std::size_t i = 0; i < 4; ++i) {
-    int ai = ma(i).get().get()();
+    int ai = ma(i).get()();
```

Comment:
This is just used for the printout in the example with the simplified API, and will have no 1to1 match in the final API.

Anyway, in my opinion your suggested interface is more problematic since it hides the synchronization of `.get()`.